helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -480,7 +480,6 @@ public long getRebalanceThrottledByErrorPartitionGauge() {
   public void resetMaxTopStateHandoffGauge() {
     if (_lastResetTime + DEFAULT_RESET_INTERVAL_MS <= System.currentTimeMillis()) {
       _maxSinglePartitionTopStateHandoffDuration.updateValue(0L);
-      _missingTopStatePartitionsBeyondThresholdGauge.updateValue(0L);
```

Comment:
Had a offline discussion. Let's check about what happens if cache gets dropped as a part of another pipeline (periodic rebalance etc) before we remove this reset part.

Reply:
Hey Rahul, Thanks for your review.  

Yes as this fix is working fine and is urgent for Espresso WAGED rollout blocker, I will address this concern in a following PR.

Reply:
please remove customer name from above.